app/test/unit/git/tag-test.ts

```diff
@@ -166,7 +166,7 @@ describe('git/tag', () => {
 
       await checkoutBranch(repository, account, branch!)
       const commitSha = await createCommit(repository, 'a commit', files)
-      await createTag(repository, 'my-new-tag', commitSha!)
```

Comment:
Since `createCommit` doesn't return `undefined` anymore, this is not needed, right? :_D

Reply:
Right!